BuildLoader.py

```diff
@@ -148,9 +131,12 @@ def __init__(self, *args, **kwargs):
 
         # Define default private key (PEM format) used to sign configuration data, firmware capsule and IAS image
         key_dir = os.path.join('BootloaderCorePkg', 'Tools', 'Keys')
-        self._CFG_PRIVATE_KEY       = os.path.join(key_dir, 'TestSigningPrivateKey.pem')
-        self._FWU_PRIVATE_KEY       = os.path.join(key_dir, 'TestSigningPrivateKey.pem')
-        self._IAS_PRIVATE_KEY       = os.path.join(key_dir, 'TestSigningPrivateKey.pem')
+        # Allow master key to be able to sign everything by default
+        self._MASTER_KEY_USAGE      = HASH_USAGE['PUBKEY_CFG_DATA'] | HASH_USAGE['PUBKEY_FWU'] | HASH_USAGE['PUBKEY_OS'] | \
+                                      HASH_USAGE['PUBKEY_CONT_DEF']
+        self._MASTER_PRIVATE_KEY    = os.path.join(key_dir, 'TestSigningPrivateKey.pem')
+        self._CFGDATA_PRIVATE_KEY   = os.path.join(key_dir, 'TestSigningPrivateKey.pem')
+        self._CONTAINER_PRIVATE_KEY = os.path.join(key_dir, 'TestSigningPrivateKey.pem')

```

Comment:
Question: 
1. Do we support IAS image verification with this? IAS image key input is removed.
2. Same for FWU_PRIVATE_KEY. How this hash is updated?

Reply:
By default, it will be verified by the master key hash.  It can be override by the  platform.